src/main/java/book/store/mybookshop/controller/AuthenticationController.java

```diff
@@ -0,0 +1,32 @@
+package book.store.mybookshop.controller;
+
+import book.store.mybookshop.dto.CreateUserRequestDto;
+import book.store.mybookshop.dto.UserDto;
+import book.store.mybookshop.dto.UserLoginRequestDto;
+import book.store.mybookshop.dto.UserLoginResponseDto;
+import book.store.mybookshop.security.AuthenticationService;
+import book.store.mybookshop.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/auth")
+@RequiredArgsConstructor
+public class AuthenticationController {
+
+    private final UserService userService;
+    private final AuthenticationService authenticationService;
+
+    @PostMapping("/register")
+    public UserDto login(@RequestBody CreateUserRequestDto userDto) {
```

Comment:
check your validation
```suggestion
    public UserDto register(@RequestBody @Valid CreateUserRequestDto userDto) {
```